test/e2e/libpod_suite_remoteclient_test.go

```diff
@@ -30,7 +30,12 @@ func SkipIfRootless() {
 
 // Podman is the exec call to podman on the filesystem
 func (p *PodmanTestIntegration) Podman(args []string) *PodmanSessionIntegration {
-	podmanSession := p.PodmanBase(args)
+	podmanSession := p.PodmanBase(args, false)
+	return &PodmanSessionIntegration{podmanSession}
+}
+
+func (p *PodmanTestIntegration) PodmanNoCache(args []string) *PodmanSessionIntegration {
```

Comment:
Can we add a comment here (and maybe update the one of `Podman()` as well)? Copy-pasting tests in IDEs will be easier.